tests/e2e/ctl_v3_role_test.go

```diff
@@ -58,94 +49,6 @@ func roleAddTest(cx ctlCtx) {
 	}
 }
 
-func rootRoleGetTest(cx ctlCtx) {
-	cmdSet := []struct {
-		args        []string
-		expectedStr interface{}
-	}{
-		// Add a role of root .
-		{
-			args:        []string{"add", "root"},
-			expectedStr: "Role root created",
-		},
-		// get root role should always return [, <open ended>
-		{
-			args:        []string{"get", "root"},
-			expectedStr: []string{"Role root\r\n", "KV Read:\r\n", "\t[, <open ended>\r\n", "KV Write:\r\n", "\t[, <open ended>\r\n"},
-		},
-		// granting to root should be refused by server
-		{
-			args:        []string{"grant-permission", "root", "readwrite", "foo"},
-			expectedStr: "Role root updated",
-		},
-		{
-			args:        []string{"get", "root"},
-			expectedStr: []string{"Role root\r\n", "KV Read:\r\n", "\t[, <open ended>\r\n", "KV Write:\r\n", "\t[, <open ended>\r\n"},
-		},
-	}
-
-	for i, cmd := range cmdSet {
-		if _, ok := cmd.expectedStr.(string); ok {
-			if err := ctlV3Role(cx, cmd.args, cmd.expectedStr.(string)); err != nil {
-				if cx.dialTimeout > 0 && !isGRPCTimedout(err) {
-					cx.t.Fatalf("roleAddTest #%d: ctlV3Role error (%v)", i, err)
-				}
-			}
-		} else {
-			if err := ctlV3RoleMultiExpect(cx, cmd.args, cmd.expectedStr.([]string)...); err != nil {
-				if cx.dialTimeout > 0 && !isGRPCTimedout(err) {
-					cx.t.Fatalf("roleAddTest #%d: ctlV3Role error (%v)", i, err)
-				}
-			}
-		}
-	}
-}
-
-func roleGrantTest(cx ctlCtx) {
-	cmdSet := []struct {
-		args        []string
-		expectedStr string
-	}{
-		// Add a role.
-		{
-			args:        []string{"add", "root"},
-			expectedStr: "Role root created",
-		},
-		// Grant read permission to the role.
-		{
-			args:        []string{"grant", "root", "read", "foo"},
-			expectedStr: "Role root updated",
-		},
-		// Grant write permission to the role.
-		{
-			args:        []string{"grant", "root", "write", "foo"},
-			expectedStr: "Role root updated",
-		},
-		// Grant rw permission to the role.
-		{
-			args:        []string{"grant", "root", "readwrite", "foo"},
-			expectedStr: "Role root updated",
-		},
-		// Try granting invalid permission to the role.
-		{
-			args:        []string{"grant", "root", "123", "foo"},
-			expectedStr: "invalid permission type",
-		},
-	}
-
-	for i, cmd := range cmdSet {
-		if err := ctlV3Role(cx, cmd.args, cmd.expectedStr); err != nil {
-			cx.t.Fatalf("roleGrantTest #%d: ctlV3Role error (%v)", i, err)
-		}
-	}
-}
-
-func ctlV3RoleMultiExpect(cx ctlCtx, args []string, expStr ...string) error {
-	cmdArgs := append(cx.PrefixArgs(), "role")
-	cmdArgs = append(cmdArgs, args...)
-
-	return e2e.SpawnWithExpects(cmdArgs, cx.envMap, expStr...)
-}
 func ctlV3Role(cx ctlCtx, args []string, expStr string) error {
```

Comment:
Line 52 and below are retained only in this PR. After auth_test are migrated to common, they should be cleaned up.